Packs/IntSight/ReleaseNotes/3_0_2.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Rapid7 - Threat Command (IntSights)
+
+- Documentation and metadata improvements.
```

Comment:
This is not just `Documentation and metadata improvements` as you fixed the outputs.